code/game/sound.dm

```diff
@@ -163,4 +163,6 @@ var/global/const/FALLOFF_SOUNDS = 0.5
 			if ("button") soundin = pick(global.button_sound)
 			if ("chop") soundin = pick(global.chop_sound)
 			if ("glasscrack") soundin = pick(global.glasscrack_sound)
+			if ("bodyfall") soundin = pick(global.bodyfall_sound)
+			if ("bodyfall_machine") soundin = pick(global.bodyfall_sound_machine)
```

Comment:
not a fan but of this

Reply:
What's the issue?

Reply:
Entire get_sfx is rude and complex to maintain. It works based on strings, not types, it requires to add new sounds into endless list. I'd prefer to move everything to `/decl/sound_repository` but this is not the PR goal.